static/bidder-info/teads.yaml

```diff
@@ -0,0 +1,9 @@
+endpoint: "https://a.teads.tv/prebid-server/bid-request"
+maintainer:
+  email: "support@teads.com"
```

Comment:
PBS team has sent verification mail in order to ensure the accuracy of maintainer  email. Requesting to respond with "received" message to confirm the email address.

Reply:
@github-maxime-liege, requesting to revert back on mail

Reply:
hey @onkarvhanumante did you send an email to the new one i updated ? 
Because i can't see anything on our side, we have a mapping email to ticket board on our side, i don't know if it is a problem for you

Reply:
@github-maxime-liege We have sent a verification email on updated email address. Please revert back with "Received" message to complete the process.
Thank you!

Reply:
Email verification completed!